app/utils/gitlab_manager.py

```
@@ -0,0 +1,45 @@
+        except Exception as e:
+            print(f"Error initializing GitLabManager: {e}")
+            self.project = None
+            self.gl = None
+            self.auth_status = False
+
+
+    def get_project(self):
+        if self.project_id !="":
+            return self.gl.projects.get(self.project_id)
+        else:
+            None
```

Ruff (0.11.9)

Found useless expression. Either assign it to a variable or remove it.

(B018)